api/graphql/query/Resource/availability.js

```diff
@@ -10,17 +10,20 @@ class ZoomAccountAvailability extends BaseResolver {
     });
     const topic = await resource.getTopic();
 
-    if (this.parent.subject_type === 'ZoomAccount')
-      return ZoomAccount.findAvailaibilityDuring(
-        this.parent.subject_id,
-        {
-          start_at: topic.start_at,
-          end_at: topic.end_at,
-        },
-        {
-          excludeTopics: [topic.id],
-        },
-      );
+    if (this.parent.subject_type === 'ZoomAccount') {
+      return (
+        await ZoomAccount.findAllAvailabilityDuring([
+          {
+            id: this.parent.subject_id,
+            dateTimeRange: {
+              start_at: topic.start_at,
+              end_at: topic.end_at,
+            },
+            excludeTopics: [topic.id],
+          },
+        ])
```

Comment:
Do you mean this?

```
await this.ctx.loaders.zoomAccountAvailabilityLoader.load({
      id: this.parent.id,
      dateTimeRange: {
        start_at: topic.start_at,
        end_at: topic.end_at,
      },
      excludeTopics: [topic.id]
    })
```

Reply:
Yes but before that lets fix the main issue .